server.py

```diff
@@ -1,25 +1,57 @@
 import bottle
 import os, sys, datetime
 import string, random
-
 from collections import defaultdict, namedtuple
-import shelve
+from peewee import *
 
 path = os.path.abspath(__file__)
 dir_path = os.path.dirname(path)
 app = bottle.Bottle()
 
-database_path = "submission_record.db"
-user_db = "user_record.db"
-sessions_db = "session_record.db"
+DATABASE_NAME = "data.db"
+
 questions = {}
 contests = {}
 question_dir = "files/questions"
 
 Question = namedtuple("Question", "output statement")
```

Comment:
we can stop using these now. Let's have tables for those

Reply:
I left questions and contests for now because I thought we would need to work with them in the next issue, and there will probably be lots of changes, so it would be better to fix them in the next one. But if we must fully switch to MySQLite before that, then I can complete them, won't take much but I would prefer to leave them for now

Reply:
Done.